### PR TITLE
docs(ootbc): added Automation Anywhere connector doc

### PR DESCRIPTION
**Pull request by Oleksiivanov**
## What is the purpose of the change

Adding a new page for the connectors related to an Automation Anywhere RPA connector.

## Are there related marketing activities

TBD.

## When should this change go live?

2023-03-14 - after https://github.com/camunda/team-connectors/issues/354

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by bastiankoerber — COMMENTED** [review]
in general looks good.
I would just check the prerequisits: They are looking really general.

Maybe check if a user needs a special license to call the api endpoints.
Or needs to configure user permission / technical apps to make it work.

if you need this, please add it there @Oleksiivanov

File: docs/components/connectors/out-of-the-box-connectors/automation-anywhere.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
+---
+id: automation-anywhere
+title: Automation Anywhere Connector
+description: Orchestrate your Automation Anywhere queue items from your BPMN process.
+---
+
+## Prerequisites
+
+The **Automation Anywhere Connector** allows you to orchestrate an Automation Anywhere queue from your BPMN process with [Automation Anywhere RPA](https://www.automationanywhere.com/).
+To start using the Connector, you have to have a running Automation Anywhere instance with actual [license](https://docs.automationanywhere.com/bundle/enterprise-v2019/page/enterprise-cloud/topics/control-room/dashboards/cloud-administration-licenses.html) we configured API service. Please refer to the [official documentation page](https://docs.automationanywhere.com/bundle/enterprise-v2019/page/enterprise-cloud/topics/aae-client/bot-creator/using-the-workbench/cloud-install.html) to learn more how to install and configure Automation Anywhere API service.
```
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/automation-anywhere.md:10]: Not exactly sure what you mean here in relation to a license. Can you explain?

```suggestion
To start using the Connector, you have to have a running Automation Anywhere instance with an a [license](https://docs.automationanywhere.com/bundle/enterprise-v2019/page/enterprise-cloud/topics/control-room/dashboards/cloud-administration-licenses.html) we configured API service. Please refer to the [official documentation page](https://docs.automationanywhere.com/bundle/enterprise-v2019/page/enterprise-cloud/topics/aae-client/bot-creator/using-the-workbench/cloud-install.html) to learn more how to install and configure Automation Anywhere API service.
```

File: docs/components/connectors/out-of-the-box-connectors/automation-anywhere.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+Also, you need a user account with the `AAE_Queue Admin` role to query and manage workload queues and work items in a Control Room. More details about roles you can find in [official documentation](https://docs.automationanywhere.com/bundle/enterprise-v2019/page/enterprise-cloud/topics/control-room/administration/roles/cloud-system-created-roles.html).
```
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/automation-anywhere.md:11]:
```suggestion
Also, you need a user account with the `AAE_Queue Admin` role to query and manage workload queues and work items in a Control Room. More details about roles you can find in the [official documentation](https://docs.automationanywhere.com/bundle/enterprise-v2019/page/enterprise-cloud/topics/control-room/administration/roles/cloud-system-created-roles.html).
```

File: docs/components/connectors/out-of-the-box-connectors/automation-anywhere.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+:::note
+All the mandatory and non-mandatory fields depending on the authentication and operation selections you choose are covered in the upcoming sections.
+:::
+
+## Authentication
+
+You can choose among the available **Automation Anywhere Connector** authentication types according to your authentication requirements.
+
+:::note
+We advise you to keep your authentications and secrets data safe and avoid exposing it in the BPMN `xml` file by creating a secret:
```
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/automation-anywhere.md:32]:
```suggestion
We advise you to keep your authentications and secrets data safe and avoid exposing it in the BPMN XML file by creating a secret:
```

File: docs/components/connectors/out-of-the-box-connectors/automation-anywhere.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+4. Select needed **Multiple login** type. If this value is set to `true`, you will be allowed multiple API sessions. For more information on multi-login, see [Multi-login user](https://docs.automationanywhere.com/bundle/enterprise-v2019/page/enterprise-cloud/topics/control-room/administration/users/cloud-multi-login-user.html).
+
+### _Authenticate (username and API key)_ authentication
+
+Select the **Automation Anywhere Connector** and fill out the following properties under the **Authentication** section:
+
+1. Select **Authenticate (username and API key)** in the **Authentication** section.
+2. Set **Password** to `Password` to the secret you created (i.e. `secrets.AUTOMATION_ANYWHERE_PASSWORD`).
+3. Set **API key** to `API key` to the secret you created (i.e. `secrets.AUTOMATION_ANYWHERE_API_KEY`). The API-Key is a 40-character string generated in the Control Room. See [create and assign API key generation role documentation](https://docs.automationanywhere.com/bundle/enterprise-v2019/page/enterprise-cloud/topics/control-room/control-room-api/cloud-control-room-apikey-role.html) for learn more.
```
Comment (igpetrov) [inline docs/components/connectors/out-of-the-box-connectors/automation-anywhere.md:53]:
```suggestion
3. Set **API key** as `API key` to the secret you created (i.e. `secrets.AUTOMATION_ANYWHERE_API_KEY`). The API-Key is a 40-character string generated in the Control Room. See [create and assign API key generation role documentation](https://docs.automationanywhere.com/bundle/enterprise-v2019/page/enterprise-cloud/topics/control-room/control-room-api/cloud-control-room-apikey-role.html) for learn more.
```

**Review by igpetrov — APPROVED** [review]
LGTM.

**Comment by igpetrov** [comment]
Hi @bastiankoerber @Oleksiivanov 

Can we update release and marketing details in PR description?

**Review by igpetrov — APPROVED** [review]

File: docs/components/connectors/out-of-the-box-connectors/automation-anywhere.md [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+id: automation-anywhere
+title: Automation Anywhere Connector
+description: Orchestrate your Automation Anywhere queue items from your BPMN process.
+---
+
+## Prerequisites
+
+The **Automation Anywhere Connector** allows you to orchestrate an Automation Anywhere queue from your BPMN process with [Automation Anywhere RPA](https://www.automationanywhere.com/).
+
+To start using the Connector, you must have a running Automation Anywhere instance with a [license](https://docs.automationanywhere.com/bundle/enterprise-v2019/page/enterprise-cloud/topics/control-room/dashboards/cloud-administration-licenses.html) we configured API service. Refer to the [official documentation page](https://docs.automationanywhere.com/bundle/enterprise-v2019/page/enterprise-cloud/topics/aae-client/bot-creator/using-the-workbench/cloud-install.html) to learn more about installing and configuring the Automation Anywhere API service.
```
Comment (christinaausley) [inline docs/components/connectors/out-of-the-box-connectors/automation-anywhere.md:11]: "To start using the Connector, you must have a running Automation Anywhere instance with a [license](https://docs.automationanywhere.com/bundle/enterprise-v2019/page/enterprise-cloud/topics/control-room/dashboards/cloud-administration-licenses.html) we configured API service."

What do you mean here? You need an instance with a license we configured via an API service?

File: docs/components/connectors/out-of-the-box-connectors/automation-anywhere.md [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+      "col6": "",
+      "col10": ""
+    }
+  ]
+}
+```
+
+![AA get work item result](../img/connector-utomation-anywhere-get-work-item-result.png)
+
+## Appendix
```
Comment (christinaausley) [inline docs/components/connectors/out-of-the-box-connectors/automation-anywhere.md:201]: I think if there is just one subject here, you can probably remove the Appendix header for now. What do you think?

**Review by christinaausley — APPROVED** [review]
Two non-blocking comments for clarity 👍 Technical review in latest commit.

**Review by igpetrov — APPROVED** [review]

File: docs/components/connectors/out-of-the-box-connectors/automation-anywhere.md (hunk 5 above)
Comment (christinaausley) [reply docs/components/connectors/out-of-the-box-connectors/automation-anywhere.md]: Thank you!

**Pull request merged**